text/survey_analysis_report2.md

```diff
@@ -0,0 +1,191 @@
+---
+title: "A folder structure to organise files in a reserch project"
+bibliography:  "GIN-tonic-paper-zotero.bib"
+author:
+  - Julien Colomb
+  - The Gin-tonic team
+output: 
+  bookdown::html_document2:
+    keep_md: true
+
+---
+
+
+![titleimage: Some positive feedback received in the survey.](../50_figures/pulication_ready_figures/blogtitle.jpeg)
+
+# Summary
+
+In contrast to software developers, researchers have no standard way to organise their files.
+While data managers recommend to design such an organisation,
+they fail at providing clear recommendations or examples;
+and there is no time allocated to this task in the researcher work.
+We believe that providing researcher with a folder tree structure template could make a huge difference in promoting data management and research collaboration.
+Here we present the results of an initial survey run in three neuroscience research consortium (LINK).
+
+
+We collected the feedback from 51 neuroscientists presented with 2 initial drafts,
+analysed their responses,
+and build a second template draft based on this feedback that is presented here.
+This new template takes three levels of data organisation into account (experiment, project, laboratory),
+while staying relatively simple and flexible, especially when implemented it in a git workflow (using the GIN-tonic tool we are developing).
+Researchers from all research domains are now invited to review this draft in a second survey (run until mid January 2021, LINK).  
+
+
+
+# Introduction
+
+Researchers spend time every day doing file management on their computers (creating, downloading, naming,  moving,  saving,  copying,  reviewing,  navigating, searching for, sharing, and deleting files and folders).
+While many different initiatives and tools tried to improve file management (using tags, databases and search algorithms), the use of folder tree structure appeared to be unavoidable [@dinneenUbiquitousDigitalFile2020]. 
+While both reproducible research and data management advocates and experts are advising researchers to design a folder organisation structure ([@LibraryCarpentryLcfairresearch2019], [@theturingwaycommunityTuringWayHandbook2019]),
+only few actually provide examples or attempt to homogenize the structure (but see [@vuorreSharingOrganizingResearch2020] and [@wilsonGoodEnoughPractices2017] for exceptions).
+
+We are data managers of different institutions working with neurobiologists, and we teamed up with the NFDI-neuro community to develop a new strategy to help researchers with their data management. 
```

Comment:
We -the authors of this blog post- are data managers of different institutions working with neurobiologists, and we teamed up with the NFDI Neuroscience community to develop a new strategy to support researchers with their data management.